src/test/groovy/net/rubygrapefruit/platform/file/OsxFileEventsTest.groovy

```diff
@@ -30,122 +30,138 @@ import spock.lang.Timeout
 class OsxFileEventsTest extends Specification {
```

Comment:
☝️ How about `@Requires({ Platform.current().macOs })` instead of `@IgnoreIf`?

Reply:
Fixed.